pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/migration/AdditionalOncHistoryDetailsRetriever.java

```diff
@@ -16,12 +16,14 @@ public class AdditionalOncHistoryDetailsRetriever extends AdditionalRecordsRetri
     }
 
     static Optional<AdditionalOncHistoryDetailsRetriever> fromRealm(String realm) {
+/*
```

Comment:
this is part of "Disables special ES export handling for OS1/OS2 ES index sharing" mentioned in the PR comments.